Backend/db/utils.js

```diff
@@ -16,8 +17,15 @@ async function decodeJwtToken(token, secret) {
   return jwt.verify(token, secret);
 }
 
+async function getCurrentUser(req) {
+  const token = req.headers["authorization"].split(" ")[1];
```

Comment:
validate is token null or not

Reply:
This will throw exception if there was no "authorization" header. Check `req.headers["authorization"]` then move on